packages/squiggle-lang/__tests__/library/danger_test.ts

```diff
@@ -23,6 +23,10 @@ describe("Danger functions", () => {
     testEvalToBe("Danger.allCombinations([3])", "[[3]]");
     testEvalToBe("Danger.allCombinations([])", "[]");
   });
+  describe("javascript", () => {
+    testEvalToBe("Danger.parseFloat('10.0') + 1", "11");
+    testEvalToBe("Danger.parseFloat('foo')", '"Parse Failed"');
```

Comment:
Wasn't sure what the best error string was, because it's in ``Danger``, we don't need to worry about quite as much.

Reply:
In the future, `parseFloat("str").unwrap()` could be good; for that, we'd need:
1. some kind of GADTs or structs
2. ideally, methods (because calling `-> Result.unwrap` is too verbose)
3. exception handling